src/components/molecules/message/message.scss

```
@@ -0,0 +1,32 @@
+@use "../../variables" as *;
+.message_outer {
+    text-align: center;
+    padding: 52px 20px;
+    @media (max-width: $breakpoint-lg) {
+        padding: 32px 20px;
+    }
+    .message_title {
+        font-family: $fontfamily-mincho;
+        font-style: normal;
+        font-weight: 400;
+        word-wrap: break-word;
```

```suggestion
        overflow-wrap: break-word;
```
> このプロパティはもともと、標準外かつ接頭辞のない word-wrap と呼ばれる Microsoft 拡張であり、多くのブラウザーはこの名前で実装していました。 overflow-wrap に改名されたため、 word-wrap は別名になりました。

出典：https://developer.mozilla.org/ja/docs/Web/CSS/overflow-wrap

上述のような経緯があるので、 `overflow-wrap` を使うようにしましょう！
テキストの中身が変わったときのことも想定できているのがいい感じです 👍